.vscode/launch.json

```diff
@@ -0,0 +1,13 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
Delete these comments please

Reply:
Sir I have not done anything in launch.json . These are auto-generated lines in vs code I think.

Reply:
remove these lines in the comment rest everything is fine